test/query/query.test.ts

```diff
@@ -97,3 +99,77 @@ describe('query/query', () => {
     });
   });
 });
+
+describe('2D aggregate', () => {
+  describe('summary with mean', () => {
+    it('should not generate 2D summary plot with MEAN(Q) x MEAN(Q1)', () => {
```

Comment:
Your test case description is still bad.  This doesn't explain why this one should not generate and why next one should.